docs/examples/advanced-configuration.md

```diff
@@ -0,0 +1,56 @@
+# Advanced configuration example
+
+The following example shows how Darkbox can be configured. In this example all configuration values use the Darkbox defaults. To learn which possible values each configuration can have please refer to the [configuration value documentation](../../README.md#Advanced-configuration).
+
+## Inclusion in the desired `.html` file
+```ts
+<darkbox-gallery [images]="images" [configuration]="configuration"></darkbox-gallery>
+```
+
+## View of the corresponding `.ts` file
+```ts
+import { Component } from '@angular/core';
+import { Image, DarkboxConfiguration } from '@failed-successfully/ngx-darkbox-gallery';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent {
+  configuration: DarkboxConfiguration = {
+      darkboxConfiguration: {
+        loopDirection: LoopDirection.BOTH,
+        enableCaption: true,
+      },
+      gridConfiguration: {
+
+      },
+      imageConfiguration: {
+          initialBatchSize = 10,
```

Comment:
All of this is the gridConfiguration. The image configuration should be empty

Reply:
Done in 0002c3f